doc/manuals/orion-api.md

```diff
@@ -1184,26 +1158,23 @@ Response:
 * Successful operation uses 200 OK
 * Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for more details.
 
-+ Parameters
-    + entityId (required, string) - Id of the entity to be retrieved
-    + type (optional, string) - Entity type, to avoid ambiguity in case there are several
-      entities with the same entity id.
-    + attrs: temperature,humidity (optional, string) - Comma-separated list of attribute names whose
-      data must be included in the response. The attributes are retrieved in the order specified by
-      this parameter. See "Filtering out attributes and metadata" section for more detail.
-      If this parameter is not included, the attributes are retrieved in arbitrary order, and all
-      the attributes of the entity are included in the response.
-    + metadata: accuracy (optional, string) - A list of metadata names to include in the response.
-      See "Filtering out attributes and metadata" section for more detail.
-    + options (optional, string) - Options dictionary
-      + Members
-          + keyValues - when used, the response payload uses the `keyValues` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + values - when used, the response payload uses the `values` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + unique - when used, the response payload uses the `values` simplified entity
-            representation. Recurring values are left out.
-            See "Simplified Entity Representation" section for details.
+**Parameters**
+
+| Parameter  | Optional | Type   | Description                                                                                                                                                                                                                                                                                                                                                                             | Example      |
+|------------|----------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| `entityId` |          | string | Id of the entity to be retrieved                                                                                                                                                                                                                                                                                                                                                        | `myRoom_112` |
```

Comment:
```suggestion
```

This table should include only query parameters. The id should be included in

```
#### Retrieve Entity [GET /v2/entities] -> #### Retrieve Entity [GET /v2/entities/{entityId}]
```

This is probably a typo in a previous PR. ToC should be also fixed